package.json

```diff
@@ -20,6 +20,8 @@
     "yarn": "1.9.4"
   },
   "dependencies": {
+    "@firebase/app-types": "^0.3.2",
+    "@firebase/database-types": "^0.3.2",
```

Comment:
why did you add those? I don't see that they're used anywhere... 😅

Reply:
They are used basically for intellisense in IDE. 
* Probably they should be placed in dev dependencies. I didn't want them installed in global.